src/sass/_certificates.scss

```diff
@@ -16,4 +16,8 @@
       display: none;
     }
   }
+
+  .download-crt {
+    margin-top: $spv--small;
+  }
```

Comment:
I guess this is to add spacing above the button? Unfortunately, it misaligns the button copy from the horizontal line spanning from the line of text to the left of it "2. Trust ... Download the .crt and ...". the button copy should be aligned with the text next to it. Maybe add the spacing to the whole row instead?

Reply:
Ahh I see, yeah okay makes sense to add margin to the entire row then